src/GitHub.Api/Helpers/SimpleJson.cs

```diff
@@ -1369,9 +1369,9 @@ public virtual object DeserializeObject(object value, Type type)
                     if (type == typeof(NPath) || (ReflectionUtils.IsNullableType(type) && Nullable.GetUnderlyingType(type) == typeof(NPath)))
                         return new NPath(str);
                     if (type == typeof(DateTime) || (ReflectionUtils.IsNullableType(type) && Nullable.GetUnderlyingType(type) == typeof(DateTime)))
-                        return DateTime.ParseExact(str, Iso8601Format, CultureInfo.InvariantCulture, DateTimeStyles.AssumeUniversal | DateTimeStyles.AdjustToUniversal);
+                        return DateTime.ParseExact(str, Constants.Iso8601Formats, CultureInfo.InvariantCulture, DateTimeStyles.AssumeUniversal | DateTimeStyles.AdjustToUniversal);
```

Comment:
I'd rather have `Iso8601Format` point to Constants instead of hardcoding that here, it's easier to maintain that way I think

Reply:
👍